packages/management-console/server/MockData/types.js

```diff
@@ -0,0 +1,150 @@
+const {gql} = require("apollo-server-express");
+module.exports = typeDefs = gql`
+    scalar DateTime
+
+    schema {
+        query: Query
+        subscription: Subscription
+    }
+
+    type ProcessInstance {
+        id: String!
+        processId: String!
+        processName: String
+        parentProcessInstanceId: String
+        rootProcessInstanceId: String
+        rootProcessId: String
+        roles: [String!]
+        state: ProcessInstanceState!
+        managementEnabled: Boolean
+        endpoint: String!
+        nodes: [NodeInstance!]!
+        error: String
```

Comment:
error will be a complex type: 

`type ProcessIntanceError {
   nodeDefinitionId: String!
   message: String
}`

```suggestion
        error: String
```

Reply:
modified